wdl/BAFFromGVCFs.wdl

```diff
@@ -215,6 +215,11 @@ task GenotypeGVCFs {
   command <<<
     set -e
 
+    # Adding this for FSx/local FS
```

Comment:
Hi @spatel-gfb - the impact of this change seems likely to be specific to the platform. It would require increasing the size of the boot disk when running with cromwell on GCP, which would increase cost and introduce another resource whose size we need to determine and which could run out of space. And, if we increased the size of the boot disk and decreased the size of the main disk, the I/O speed would decrease, since I/O improves with increasing disk size on GCP. Due to the potential for a regression for GCP, we have concerns about merging this change and would prefer to retain the current behavior. It may be necessary going forward for your AWS fork to differ slightly in cases like this.

Reply:
@epiercehoffman : would adding the shared_filesystem flag just for these processes work for you? Coz by default it will be false, but in case of AWS or other teams wanting to use local, can enable it ?

Reply:
@epiercehoffman : any updates or comments on this ?

Reply:
@mwalker174 We already raised this issue with GATK ream : https://github.com/broadinstitute/gatk/issues/7646 coz its pertaining to genomicsdb tool and not cromwell. There suggestions still didnt improve the performance and the latest suggestion would need to be implemented on the pipeline level and hence we haven't test it yet and would need your teams help to test it out first.

The genomicsdb tool itself has performance issues with posix type of FS. Hence were using this workaround. Initially as per Emma's suggestion we added shared_fs flag. That would default be False (no changes ideally would be needed to run on GCP) but for AWS, we can pass it as true to handle it or even on GCP if someone is using posix type of FS.

We still think using that flag just for this and MELT wdl would be really helpful and get this out sooner instead of waiting few months for the newer/updated version of pipeline.

We welcome your inputs and would like to follow whatever is best for both the platforms.